bittide/src/Bittide/Wishbone.hs

```diff
@@ -103,6 +107,13 @@ ilaWb ::
     (Wishbone dom 'Standard addrW a)
 ilaWb = Circuit $ \(m2s, s2m) ->
   let
```

Comment:
I´d consider enabling the user to change the `stages` and `depth` field of `IlaConfig`.
Either by passing those as argument or allowing `IlaConfig` as argument and adjusting the fields internally. (with a check ofc)

Reply:
Done, does this look good to you?